apps/www/_blog/2024-12-06-database-build-v2.mdx

```diff
@@ -0,0 +1,218 @@
+
+![database.build v2 Deploy to Supabase](/images/blog/launch-week-13/day-5-database-build-v2/deploy-to-supabase-dark.png)
+
+Yes, it’s a lot of moving parts! But this approach let us deliver cloud deployments today while we wait for WASM `pg_dump` to be production-ready. Kudos to the ElectricSQL team for continually breaking new ground with embeddable Postgres. We can’t wait to simplify this process even further in the future.
+
+### What about serverless PGlite?
+
+Back when we first launched database.build, we shared plans to explore a serverless PGlite deployment option. However as we moved forward, we encountered a number of small but significant challenges that added up over time. PGlite’s memory usage in multi-tenant environments turned out to be higher than anticipated—a problem the ElectricSQL team is actively working to address. Given PGlite’s single-connection limit, anything more than a few megabytes of RAM will not be practical in a serverless environment. Additionally, our initial reliance on `s3fs` for storage proved to be less reliable than expected, making it difficult to meet the requirements for a seamless serverless experience.
+
+While these challenges have pushed us to focus on alternative deployment options for now, we remain committed to revisiting serverless options as these issues are resolved and the ecosystem continues to evolve.
+
+## Drag and drop SQL files
+
+You’ve always been able to drag and drop CSV files into the chat, but what about SQL files? In this new release, simply dropping a SQL file onto that chat will automatically load its contents into your browser DB.
+
+<video width="99%" autoPlay loop muted playsInline controls={true}>
+  <source
+    src="/images/blog/launch-week-13/day-5-database-build-v2/drag-drop-sql-clip.mp4"
+    type="video/mp4"
+  />
+</video>
+
+<video width="99%" autoPlay loop muted playsInline controls={true}>
+  <source
+    src="/images/blog/launch-week-13/day-5-database-build-v2/drag-drop-sql-direct-drop.mp4"
+    type="video/mp4"
+  />
+</video>
+
+This can be useful if you have existing schema or migrations that were designed elsewhere that you wish to extend or ask questions about.
+
+## Mobile support
+
+We’re excited to share that database.build now works on mobile! Whether you’re on the train, stuck in a meeting, or lounging on the couch, you can spin up databases and tweak your schema with just a few taps.
+
+![database.build v2 mobile support](/images/blog/launch-week-13/day-5-database-build-v2/mobile-support.png)
+
+With mobile support, you officially have no excuse not to ship that feature you’ve been procrastinating on. We’re excited to see what you build!
+
```

Comment:
@gregnr i cut out the `Redesign` section b/c it felt redundant given that the other vids and screenshots already contained the new redesign and i used the mobile image above in the `Mobile support` section.

feel free to add it in though.